openquakeplatform/openquakeplatform/test/utils.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import time
+import sys
+
+from selenium import webdriver
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+
+from selenium.common.exceptions import StaleElementReferenceException
+
```

Comment:
I see a lot of imported stuff which is not used below, pyflakes should complain about that.